### PR TITLE
Home Task CSS_03

### PR DESCRIPTION
**Pull request by IrynaSafonava**
Good day Siarhei,
Didn't cope with additional home task. Could you please give some tips or promts ?
Thank you !

### CODE REVIEW

**Comment by pluhin** [comment]
> Didn't cope with additional home task. Could you please give some tips or promts ?

Which one? 
flex - you have file-tips on our site
for chess -  
```
tr:nth-child(odd) td:nth-child(even),
tr:nth-child(even) td:nth-child(odd) {}
```

navigation one from the many
```
...
li ul {
	        	position: absolute;
	        	top: 0;
	        	left: 149px;
	        	transition:visibility 0s 1s,opacity 1s; 
    			visibility:hidden; 
    			opacity:0; 

			}
	      	li:hover ul {
	      		opacity:0.9; 
        		visibility:visible; 
        		transition-delay:.1s;
	      	}
...
```

File: isafonova/CSS_02/CSS_lists.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		<style type="text/css">
+			.list {width: 350px}
+			ul {list-style: upper-roman}
+			li ul {list-style: lower-roman}
+			.python {
+				width: 400px;
+				background-color: black;
+				color: green;
+				line-height: 1rem;
+				max-height: 4em; /*still getting the scroll line, it couldn't fit 4 lines w/o scrolling*/
```
Comment (pluhin) [inline isafonova/CSS_02/CSS_lists.html:15]: 4rem

**Comment by IrynaSafonava** [comment]
Thank you Siarhei,
Yes, I meant an extra task CSS_3 where it needs to perform an effect of a smooth appearance submenus for the basic task. Now I can see transition: and visibility: need to be implemented somehow. 
Will you approve this PR w/o extra task or there is a way to cancel current Pull request in order to add missing task and then create another PR ?

**Review by pluhin — APPROVED** [review]
> Thank you Siarhei,
> Yes, I meant an extra task CSS_3 where it needs to perform an effect of a smooth appearance submenus for the basic task. Now I can see transition: and visibility: need to be implemented somehow.
> Will you approve this PR w/o extra task or there is a way to cancel current Pull request in order to add missing task and then create another PR ?

You can add this task on next PR

File: isafonova/CSS_02/CSS_lists.html (hunk 1 above)
Comment (IrynaSafonava) [reply isafonova/CSS_02/CSS_lists.html:15]: Corrections have been implemented with the latest PR today, but It didn't work for me. Perhaps I misunderstood. Still showing 3.9 rows and to see 0.1 row it needs to scroll a little.

**Pull request merged**